commit ee0445d6b6e9ec9149aa2d1b5b89991d0829883e
Author: Lukas Bulwahn <lukas.bulwahn@redhat.com>
Date:   Mon Feb 17 11:06:43 2025 +0100

    MAINTAINERS: adjust entries in FORTIFY_SOURCE and KERNEL HARDENING
    
    Commit db6fe4d61ece ("lib: Move KUnit tests into tests/ subdirectory") adds
    a file entry to the non-existing file scripts/test_fortify.sh. Probably,
    this entry intends to refer to ./lib/test_fortify/test_fortify.sh, though.
    As that file is already covered by the entry lib/test_fortify/*, there is
    no need for a separate file entry. So, drop the unnecessary file entry to
    the test_fortify script.
    
    Further, this commit misses to adjust the entry referring to
    lib/usercopy_kunit.c, which is moved to lib/tests. So, also adjust that
    file entry.
    
    Fixes: db6fe4d61ece ("lib: Move KUnit tests into tests/ subdirectory")
    
    Signed-off-by: Lukas Bulwahn <lukas.bulwahn@redhat.com>
    Link: https://lore.kernel.org/r/20250217100643.20182-1-lukas.bulwahn@redhat.com
    Signed-off-by: Kees Cook <kees@kernel.org>

diff --git a/MAINTAINERS b/MAINTAINERS
index 78be36fb1a74..8f37e238bf55 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -9068,7 +9068,6 @@ F:	include/linux/fortify-string.h
 F:	lib/test_fortify/*
 F:	lib/tests/fortify_kunit.c
 F:	lib/tests/memcpy_kunit.c
-F:	scripts/test_fortify.sh
 K:	\bunsafe_memcpy\b
 K:	\b__NO_FORTIFY\b
 
@@ -12588,7 +12587,7 @@ F:	arch/*/configs/hardening.config
 F:	include/linux/overflow.h
 F:	include/linux/randomize_kstack.h
 F:	kernel/configs/hardening.config
-F:	lib/usercopy_kunit.c
+F:	lib/tests/usercopy_kunit.c
 F:	mm/usercopy.c
 F:	security/Kconfig.hardening
 K:	\b(add|choose)_random_kstack_offset\b